src/main/java/org/digma/intellij/plugin/insights/AbstractInsightsMessageRouterHandler.java

```diff
@@ -86,6 +90,18 @@ private void pushInsightsListData(JsonNode jsonNode) {
         InsightsService.getInstance(project).refreshInsightsList(backendQueryParams);
     }
 
+    private void dismissInsight(JsonNode jsonNode) throws JsonProcessingException {
+        Log.log(LOGGER::debug, project, "got INSIGHTS/DISMISS message");
+        var insightId = getObjectMapper().readTree(jsonNode.get("payload").toString()).get("insightId").asText();
```

Comment:
there are two common methods getPayloadFromRequest and getPayloadFromRequestNonNull,
please see examples and use them